examples/checkout/sdk-switch-network-with-nextjs/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "sdk-connect-with-nextjs",
```

Comment:
Examples should all start with @examples/ for their package.json name. Maybe something like `@examples/sdk-connect-with-nextjs`?